library/Fission/Web/User/UpdateExchangeKeys.hs

```diff
@@ -0,0 +1,33 @@
+module Fission.Web.User.UpdateExchangeKeys
+  ( API
+  , server
+  ) where
+
+import           Servant
+
+import           Fission.Prelude
+import           Fission.Authorization
+ 
+import qualified Fission.Web.Error as Web.Error
+
+import qualified Crypto.PubKey.RSA as RSA
+import qualified Fission.User      as User
+
+type API
+  =  Summary "Update Public Exchange Keys"
+  :> Description "Set currently authenticated user's root list of public exchange keys"
+  :> ReqBody '[JSON] [RSA.PublicKey]
```

Comment:
Legitimate question:is the user allowed to update their exchange keys to `[]`?

Reply:
Are they able to? Yes
Should they be able to? I'm not totally sure :thinking: Updating to `[]` is essentially the same thing as "deleting" your exchange keys (which we don't have a route for yet) I can imagine a use case for deleting your keys, but it seems very uncommon

Reply:
what I posted lower would address this:
> I'm actually wondering if this route would make sense split into 2:
> - one for adding an exchange key
> - one for removing an exchange key